pyproject.toml

```diff
@@ -13,9 +13,9 @@ authors = [
     { name = "Pablo Le Henaff", email="pablo.le-henaff@pasqal.com" }
 ]
 requires-python = ">=3.10"
-license = {text = "Proprietary"}
+license = {text = "MIT-derived"}
```

Comment:
this will also need to be done in the `pyproject.toml` files buried in the `ci` directory. There's two of them.